torch/_export/serde/serialize.py

```diff
@@ -406,7 +408,8 @@ def handle_placeholder(self, node: torch.fx.Node):
         elif isinstance(node.meta["val"], torch.SymInt):
             raise AssertionError("SymInt graph input is not implemented yet.")
         elif isinstance(node.meta["val"], (int, bool, str, float, type(None))):
-            graph_input = self.serialize_input(node.meta["val"])
+            graph_input = Argument.create(as_constant=ConstantArgument(name=node.name))
+            self.graph_state.constant_values[node.name] = self.serialize_input(node.meta["val"])
```

Comment:
Hmm, I think the name "ConstantArgument" is a little confusing because the graph signature also includes [ConstantArgument](https://fburl.com/code/g325pe9u) which refer to inputs that are constants (ints, floats, strings). But here your ConstantArgument seems to refer to the constant *tensor* arguments. 

But also, I'm not sure if we need the concept of a "ConstantArgument" because they should just be TensorArguments, just like how we don't differentiate between like a "ParameterArgument" or "BufferArgument"

Reply:
Ohh this is actually the constant (int/float/str) case - currently they're being saved with just the concrete value, and no name attached, so we're unable to recover them when deserializing. But I think after talking to to @zhxchen17 the implementation will be changed...